kumascript/macros/DeprecatedGeneric.ejs

```diff
@@ -41,15 +35,7 @@ if ($1) {
         ver = "HTML";
 
         if (string.length(N) > 0) {
-            if(wiki.pageExists('/' + lang + '/docs/HTML/HTML' + N)) {
-                link = '/' + lang + '/docs/HTML/HTML' + N;
-            } else if(wiki.pageExists('/en-US/docs/HTML/HTML' + N)) {
-                link = '/en-US/docs/HTML/HTML' + N;
-            } else if(wiki.pageExists('/' + lang + '/docs/HTML')) {
-                link = '/' + lang + '/docs/HTML';
-            } else {
-                link = '/en-US/docs/HTML';
-            }
+            link = '/en-US/docs/HTML';
```

Comment:
Ditto. Shouldn't this become:
```
link = '/' + lang + '/docs/HTML/HTML' + N;
```
because the first `if` statement would always be true.

Reply:
Yeah, agreed. I'm not sure if the closing else statement is necessary since `if (string.length(N) > 0)` will always be true, so I removed it.